client/src/components/Icon/Icon.js

```diff
@@ -7,7 +7,8 @@ import React from 'react';
 const Icon = ({ name, className, title }) => (
   <span>
     <svg className={`icon icon-${name} ${className || ''}`} aria-hidden="true">
-      <use href={`#icon-${name}`}></use>
+      {/* TODO: Should not hardcode the admin path ... */}
```

Comment:
😄  a new attribute for https://github.com/wagtail/wagtail/blob/master/wagtail/admin/templates/wagtailadmin/admin_base.html#L32?